admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -364,13 +409,13 @@
         const selectedFile = this.selectedFiles[0];
 
         if (
-          this.coverData.file
-          && this.selectedFiles.length === 1
-          && this.coverData.file.image_id === selectedFile.image_id
+          this.selectedFiles.length === 1
         ) {
-          selectedFile.is_cover = this.coverData.value;
+          selectedFile.is_cover = isCover;
         }
 
+        selectedFile.legends = captionValue;
```

Comment:
One problem here is that if the save request fails, the global modal will already have been updated Should we update it only after the saving? (so the `saveImageInformations` parameter should not be the `selectedFile` but rather `captionValue` and `isCover`)

Or do you think it's ok and we should keep the model as the merchant expects it to be even if the request fails so that he doesn't lose the modifications? Both behaviour are valid anyways

Reply:
I think we shouldn't remove the text, overwise merchants could lose important writing works :/

So we update the local model, and we send an error in order to provide them a feedback about what wrongly happened

Reply:
Ok, good for me as well Just wanted to be sure about the expected behaviour